content/2_Bronze/Intro_DS.mdx

```diff
@@ -160,6 +160,18 @@ dynamic arrays (e.g. `vector<vector<int>>`) static arrays of dynamic arrays
 `vector<array<int,5>>`), and so on.
 
 <!-- Usually, the choice between a static array and a dynamic array is just personal preference. -->
```

Comment:
wait why is this commented out lol

Reply:
I don't know it was commented out before hand

Reply:
@maggie-j-liu think you can answer?

Reply:
uhh wasn't me that commented it out, according to git blame it was @bqi343

Reply:
I don't see how including this line adds any value.

Reply:
then delete it?

Reply:
sure you can if you want to

Reply:
Ok, Deleted it